katas/content/KatasLibrary.qs

```diff
@@ -6,43 +6,21 @@ namespace Microsoft.Quantum.Katas {
     open Microsoft.Quantum.Diagnostics;
     open Microsoft.Quantum.Math;
     open Microsoft.Quantum.Random;
-
-    /// # Summary
-    /// Given two operations, checks whether they act identically for all input states.
-    /// This operation is implemented by using the Choi–Jamiołkowski isomorphism.
-    operation CheckOperationsEquivalence(
-        op : (Qubit[] => Unit is Adj + Ctl),
-        reference : (Qubit[] => Unit is Adj + Ctl),
-        inputSize : Int)
-    : Bool {
-        Fact(inputSize > 0, "`inputSize` must be positive");
-        use (control, target) = (Qubit[inputSize], Qubit[inputSize]);
-        within {
-            EntangleRegisters(control, target);
-        }
-        apply {
-            op(target);
-            Adjoint reference(target);
-        }
-
-        let areEquivalent = CheckAllZero(control + target);
-        ResetAll(control + target);
-        areEquivalent
-    }
+    open Microsoft.Quantum.Diagnostics;
```

Comment:
You shouldn't need to re-open this namespace, it's already open in line 6

Reply:
I will setup my local environment soon! I haven't done that yet

Reply:
When running my build I am getting this error. have you seen this before? all of my checks are passing. I am also on main.

```
Checking formatting...
All matched files use Prettier code style!
Running the cargo fmt and clippy checks
    Checking qsc_frontend v0.0.0 (/Users/aragon/Projects/qsharp/compiler/qsc_frontend)
error: you should put bare URLs between `<`/`>` or make a proper Markdown link
   --> compiler/qsc_frontend/src/resolve.rs:130:35
    |
130 |     /// it is missed in the list. https://github.com/microsoft/qsharp/issues/897
    |                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
```

Reply:
Hi @goshua13, looks like the most recent Rust version enhanced some clippy lints, and our scripts treat warnings as errors by default to help keep the code warning clean. I'll open a PR with fixes for this soon! In the meantime, you can build locally with `--no-checks` to skip this clippy check step of the build script.

Reply:
See #1492

Reply:
got it! thanks